src/status_im2/integration_test/core.cljs

```diff
@@ -0,0 +1,58 @@
+(ns status-im2.integration-test.core
+  (:require
+    [cljs.test :refer [deftest is]]
+    [day8.re-frame.test :as rf-test]
+    [re-frame.core :as rf]
+    [status-im.events]
+    [status-im.multiaccounts.logout.core :as logout]                                   ; so integration tests can run independently
+    [status-im.subs.root]
+    [status-im.utils.test :as utils.test]
+    [status-im2.events]
+    [status-im2.integration-test.chat]
+    [status-im2.integration-test.constants :as constants]
+    [status-im2.integration-test.helpers :as helpers]
+    [status-im2.integration-test.wallet]
+    [status-im2.navigation.core]                                  ; so integration tests can run independently
+    [status-im2.subs.root]
```

Comment:
Just `status-im2.subs.root` is possible (without square brackets) because it's not being aliased.